package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@r2d2bzh/moleculer-start-checker-middleware",
+  "version": "0.0.0",
+  "description": "Checker for moleculer that checks that services are started",
+  "main": "index.js",
+  "scripts": {
+    "pretest": "npm install",
+    "test": "ava",
+    "pretest:debug": "npm run pretest",
+    "test:debug": "ava debug --host 0.0.0.0"
```

Comment:
`--host 0.0.0.0` is only necessary for instance in a Docker container to be able to publish the debug port on the host, this is not needed here (no docker or other use case).